### PR TITLE
Describe volume

### PR DESCRIPTION
**Pull request by kkg-portworx**
Following are the changes

- Added describe volume command
- Lots of common code (between get volumes and describe volume) moved to pkg/portworx
- Added UTs for the common code I added in pkg/portworx. As part of this added a json file for creating objects in the pkg/portworx. This is a large file but is mainly a data file, so can be ignored during review.
- Added UTs for describe volume.  
- Added pkg/openstorage/proto/time/prototime.go and pkg/openstorage/sched/intervals.go. These 2 files are from libopenstorage.
- Added github.com/dustin/go-humanize to vendor
- Modified copyright to Portworx

So the main files that need review are

- cmd/getVolumes.go
- cmd/describe.go 
- cmd/describeVolume.go 
- cmd/volumeFormatter.go
- pkg/portworx/pxVolumeOps.go
- pkg/portworx/volumeCommon.go

UTs added are

- cmd/describeVolume_test.go
- pkg/portworx/pxVolumeOps_test.go
- pkg/portworx/volumeCommon_test.go

### CODE REVIEW

File: pkg/proto/time/prototime.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import (
+	"time"
+
+	google_duration "github.com/golang/protobuf/ptypes/duration"
+	google_protobuf "github.com/golang/protobuf/ptypes/timestamp"
+)
+
+// TimeToTimestamp converts a go Time to a protobuf Timestamp.
+func TimeToTimestamp(t time.Time) *google_protobuf.Timestamp {
```
Comment (lpabon) [inline pkg/proto/time/prototime.go]: I think ptypes has most of these functions already. We use them in OpenStorage [API](https://github.com/libopenstorage/openstorage/blob/8be8bb8b90578ff2cfa84e1e90958df61f64dc9b/api/api.go#L912)

**Comment by lpabon** [comment]
Are some of the files copied from another project?

File: pkg/proto/time/prototime.go (hunk 1 above)
Comment (kkg-portworx) [reply pkg/proto/time/prototime.go]: These 2 files are copied from libopenstorage.

**Comment by kkg-portworx** [comment]
2 files 

- pkg/proto/time/prototime.go 
- pkg/sched/intervals.go 

are copied from libopenstorage

File: cmd/common_test.go [hunk 2]
```diff
@@ -119,3 +119,19 @@ func testCreateClone(t *testing.T, volId string, cloneName string) string {
...
+	for _, v := range volNames {
+		cli = fmt.Sprintf("%v %v", cli, v)
+	}
+	lines := executeCli(cli)
+	l := strings.Join(lines, "\n")
+	vols := strings.Split(l, "\n\n")
+	// There will be an empty last one in addition to the 3 volumes due to all
+	// of the split and join and again split
+	assert.Equal(t, len(vols), 4, "Num vols not matching")
+	// Remove the last one as it is spurious
```
Comment (lpabon) [inline cmd/common_test.go]: Not now, but later, can you add some comments on why it is spurious? Example output maybe?

**Comment by lpabon** [comment]
Part of Issue #18

**Pull request merged**